corehq/apps/hqwebapp/uglify.py

```diff
@@ -84,7 +84,7 @@ def output(self, mode='file', forced=False):
                 settings.STATIC_ROOT, self.output_dir, self.output_prefix
             )
             if not os.path.exists(js_compress_dir):
-                os.makedirs(js_compress_dir, 0775)
+                os.makedirs(js_compress_dir, 775)
```

Comment:
makedirs is supposed to be an octal https://docs.python.org/2/library/os.html https://docs.python.org/3/library/os.html

Reply:
👍 updated